### PR TITLE
Implement Sign In Screen UI

### LINKED ISSUE #7: Implement Sign In Screen UI
The Sign in screen UI is ready

Image

### PR DESCRIPTION
**Pull request by truongtdph40209**
## Related issues
Resolved #7 

## Summary
Implement Sign In Screen UI

## How to test
Run app

## Screenshots
image

### CODE REVIEW

File: components/atoms/InputCustom.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import { Input, type InputProps, Text, XStack, YStack } from 'tamagui'
+
+import getColors from '~/constants/Colors'
+
+type props = {
+  label: string
+  icon?: JSX.Element
+} & InputProps
+
+const InputCustom: React.FC<props> = (props: props) => {
```
Comment (NguyenVanViet2004) [inline components/atoms/InputCustom.tsx:12]: Rename this component

File: components/atoms/NegativeButton.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  return (
+    <Button
+      {...props}
+      fontWeight={'500'}
+      fontSize={18}
+      borderWidth={0}
+      borderRadius={50}
+      icon={props.icon}>
+
+      {props.title}
```
Comment (NguyenVanViet2004) [inline components/atoms/NegativeButton.tsx:19]: Use title property

File: components/atoms/SafeArea.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  children: React.ReactNode
+  style?: ViewStyle
+}
+
+const SafeArea = (props: Props): JSX.Element => {
+  const colors = getColors(useColorScheme())
+
+  return (
+    <SafeAreaView
+      style={[props.style, { backgroundColor: colors.lightSilver }]}
```
Comment (NguyenVanViet2004) [inline components/atoms/SafeArea.tsx:16]: Don't set background here

File: components/molecules/InputForm.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+                    onChangeText={text => { setPassword(text) }}
+                    icon={
+                        <Button
+                            unstyled
+                            justifyContent="center"
+                            right={40}
+                            onPress={togglePass}>
+                            {
+                                showPass
+                                    ? (<Feather name="eye-off" size={24} />)
```
Comment (NguyenVanViet2004) [inline components/molecules/InputForm.tsx:47]: Use onPress of Expo icon

File: components/molecules/InputForm.tsx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+                />
+            </YStack>
+            <Button unstyled>
+                <Text
+                    fontWeight="400"
+                    fontSize={13}
+                    textAlign="right"
+                    color={colors.slateGray}
+                    marginTop={10}>
+
```
Comment (NguyenVanViet2004) [inline components/molecules/InputForm.tsx:61]: Remove spacing

File: components/templates/SignInTemplate.tsx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  const handleBack = (): void => {
+  }
+  return (
+    <SafeArea style={styles.container}>
+      <Header
+        title={t('helloAgain')}
+        subtitle={t('welcomeBackYouveBeenMissed')}
+        iconLeft={
+          <NegativeButton
+            icon={<AntDesign name="left" size={18} color={colors.black} />}
```
Comment (NguyenVanViet2004) [inline components/templates/SignInTemplate.tsx:26]: It should be `start`

File: components/atoms/InputCustom.tsx (hunk 1 above)
Comment (truongtdph40209) [reply components/atoms/InputCustom.tsx:12]: ok

File: components/atoms/SafeArea.tsx (hunk 3 above)
Comment (truongtdph40209) [reply components/atoms/SafeArea.tsx:16]: yess

File: components/molecules/InputForm.tsx (hunk 4 above)
Comment (truongtdph40209) [reply components/molecules/InputForm.tsx:47]: done

File: components/atoms/EditableField.tsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+import { Input, type InputProps, Text, XStack, YStack } from 'tamagui'
+
+import getColors from '~/constants/Colors'
+
+type props = {
+  label: string
+  icon?: JSX.Element
+} & InputProps
+
+const EditableField: React.FC<props> = (props: props) => {
```
Comment (NguyenVanViet2004) [inline components/atoms/EditableField.tsx:12]: What is this? This name for component is so bad

File: components/atoms/EditableField.tsx [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+      <XStack>
+        <Input
+          {...props}
+          paddingHorizontal={16}
+          borderRadius={50}
+          height={48}
+          backgroundColor={colors.white}
+          alignItems="center"
+          width={'100%'}
```
Comment (NguyenVanViet2004) [inline components/atoms/EditableField.tsx:32]: width='100%'

**Review by NguyenVanViet2004 — APPROVED** [review]

**Pull request merged**